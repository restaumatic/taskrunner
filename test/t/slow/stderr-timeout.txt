# Similar to `stdout-timeout`, but only stderr is held up.
sleep 2 >/dev/null &
disown
