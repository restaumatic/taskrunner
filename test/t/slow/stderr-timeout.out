-- output:
[toplevel] warn | taskrunner: Task did not close stderr 1 seconds after exiting.
[toplevel] warn | Perhaps the file descriptor was leaked to a background process?
[toplevel] warn | Build will continue despite this error, but some output may be lost.
